src/main/java/uk/nhs/digital/nhsconnect/lab/results/translator/mapper/SpecimenMapper.java

```diff
@@ -56,12 +55,16 @@ private Specimen edifactToFhir(final SpecimenDetails edifact, final Patient pati
                 .setValue(identifier)
                 .setSystem(ACCESSION_IDENTIFIER_SYSTEM)));
         // fhir.status = [none]
-        // fhir.type = SG16.SPC.C832.7866
-        Optional.ofNullable(edifact.getCharacteristicType())
-            .map(SpecimenCharacteristicType::getTypeOfSpecimen)
-            .ifPresent(specimenType -> fhir.getType().addCoding()
-                .setDisplay(specimenType)
-                .setSystem(SNOMED_CODING_SYSTEM));
+        // fhir.type = SG16.SPC.C832
+        Optional.ofNullable(edifact.getCharacteristic())
+            .ifPresent(specimenCharacteristic -> {
+                final var coding = fhir.getType().addCoding()
+                    .setSystem(SNOMED_CODING_SYSTEM);
```

Comment:
Do we know if the system is the same for different characteristic types? Any idea what `903` means and if this affects the system in any way?

Reply:
No clue!

Reply:
As we couldn't find this information, we've agreed to keep it as it is and if there's a problem then hopefully the test partners will find it!